src/view-compiler.js

```diff
@@ -158,7 +158,7 @@ export class ViewCompiler {
       return this._compileElement(node, resources, instructions, parentNode, parentInjectorId, targetLightDOM);
     case 3: //text node
       //use wholeText to retrieve the textContent of all adjacent text nodes.
-      let expression = resources.getBindingLanguage(this.bindingLanguage).inspectTextContent(resources, node.wholeText);
+      let expression = resources.getBindingLanguage(this.bindingLanguage).inspectTextContent(resources, node.wholeText || node.data);
```

Comment:
I sent the PR: https://github.com/tmpvar/jsdom/pull/1652

I was thinking you could do something like this in templating/view-compiler.js while we wait for the jsdom release:

```js
import {DOM} from 'aurelia-pal';

const textAccessorProperty = DOM.createTextNode('test').wholeText === undefined ? 'textContent' : 'wholeText';

export class ViewCompiler {
   ....
```
then `...inspectTextContent(resources, node.wholeText)` would change to `...inspectTextContent(resources, node[textAccessorProperty])`